portal-kernel/src/com/liferay/portal/kernel/upload/UploadServletRequestSettingsUtil.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.kernel.upload;
+
+import com.liferay.portal.kernel.util.ServiceProxyFactory;
+
+/**
+ * @author Pei-Jung Lan
+ */
+public class UploadServletRequestSettingsUtil {
```

Comment:
The commit sequence is out of order in the pull request. Looks like it's showing the commit in chronological order. I used git rebase to reorder the commits. I'll reorder it with cherry-pick.